pages/linux/runuser.md

```diff
@@ -0,0 +1,23 @@
+# runuser
+
+> Run commands as a specific user and group IDs. Does not ask for password. Only useful when run as the root user.
+
+- Run command as a different user:
+
+`runuser {{user}} -c '{{command}}'`
+
+- Run command as a different user and group:
+
+`runuser {{user}} -g {{group}} -c '{{command}}'`
+
+- Start a login shell as a specific user:
+
+`runuser {{user}} -l`
+
+- Specific a shell for running or login instead of default shell:
+
+`runuser {{user}} -l -s /bin/sh`
+
+- Preserve the entire environment of root (if running by root). Ignore when having option `--login`:
```

Comment:
```suggestion
- Preserve the entire environment of root (only if `--login` is not specified):
```
Simplified a bit, since the command can basically only be run by root.

Reply:
Great. Thanks for reviewing.